exposed-money/src/test/kotlin/org/jetbrains/exposed/sql/money/MoneyTests.kt

```diff
@@ -107,11 +159,27 @@ open class MoneyBaseTest : DatabaseTestsBase() {
             assertEquals(toInsert, inserted)
         }
     }
+
+    private fun testInsertedAndSelectByComponentColumns(toInsert: Money?) {
+        withTables(Account) {
```

Comment:
It's minor, but I'm usually trying to call `withTables`/`withDb` on the root test function, mostly because of `excludeSettings` to make it clear, which DBs are involved.

And usually, if I do `assert()` inside function, I call this function also `assertSomething()` to show that it may stop the test

Reply:
I agree that it is nicer when tests easily make it clear from the start what tables are being used, as well as what db are being excluded. I'll change all the old code.